forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt

```diff
@@ -0,0 +1,13 @@
+Name:Lonis, Genetics Expert
+ManaCost:1 GU GU
+Types:Legendary Creature Snake Elf Detective
+PT:1/2
+K:Evolve
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigInvestigate | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME investigate that many times.
```

Comment:
use NICKNAME

Reply:
Cool - didn't know that existed. Updated.